src/trace/implementations/ord_neu.rs

```diff
@@ -159,7 +159,6 @@ mod val_batch {
     ///
     /// The `L` parameter captures how the updates should be laid out, and `C` determines which
     /// merge batcher to select.
-    #[derive(Abomonation)]
```

Comment:
Does this need a `#[derive(Serialize, Deserialize)]`? I don't know, and maybe we wont until we start to use it? :)

Reply:
We cannot easily. Serde requires the GAT to be serialize/deserialize unless we tell it otherwise.

Reply:
Gotcha. And .. no particular reason to want to serialize / deserialize these, outside perhaps of interest in some DD-based persistence? I guess I forget why the `derive(Abomonation)` was there in the first place, and perhaps there was no reason anymore?